cornflow-server/cornflow/schemas/execution.py

```diff
@@ -114,6 +114,10 @@ def get_indicators(self, obj):
     indicators = fields.Method("get_indicators")
 
 
+class ExecutionDetailsWithIndicatorsAndLogResponse(ExecutionDetailsEndpointWithIndicatorsResponse):
+    basic_log_json = fields.Nested(BasicLogSchema, attribute="log_json")
```

Comment:
Why not explode this and give back the two or three fields that we want as new fields instead of a dict inside a dict?

This can be done with a simple `fields.Method(callable)` where callable is the function that extracts the desired value from the log that can be defaulted to some other value in case it is needed.

Reply:
I did it the same as the ExecutionLogEndpointResponse schema, if log_json is a json it seems cleaner to me to parse it with a schema than to make a method that extracts three fields. In case the log schema was changed at some point, that would be enough and there would be no need to go to the method to change the fields.